### PR TITLE
Update KMG01 after `g_km` refactor

### LINKED ISSUE #238: Updates after `g_km` refactor
Update KMG01 following `g_km` refactor in `tern` (https://github.com/insightsengineering/tern/pull/1210)

### PR DESCRIPTION
**Pull request by edelarua**
Closes #238

### CODE REVIEW

**Comment by edelarua** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by edelarua** [comment]
Development will be fixed by https://github.com/insightsengineering/tern/pull/1215

File: book/graphs/efficacy/kmg01.qmd [hunk 1]
```diff
@@ -86,11 +152,13 @@ plot <- g_km(
...
 plot
 ```
+:::
 
 ## Plot Modifying <br/> Censoring Marks
 
+::: {.content-visible when-profile="development"}
 <!-- skip strict because of https://github.com/r-lib/gtable/pull/94 -->
-```{r plot5, fig.height = 8, test = list(plot_v5 = "plot"), opts.label = "skip_test_strict"}
+```{r plot5, fig.width = 9, fig.height = 6, test = list(plot_v5 = "plot"), opts.label = "skip_test_strict"}
```
Comment (Melkiades) [inline book/graphs/efficacy/kmg01.qmd:161]: why not plot5-stbl?

Comment (Melkiades) [reply book/graphs/efficacy/kmg01.qmd:161]: ok it is development

**Review by Melkiades — APPROVED** [review]
LGTM! The output plots are so much more profi, I reiterate. Thanks Emily!! Terrific job :)

**Pull request merged**